### ISSUE TITLE
UG/DG pdf submission

### ISSUE THREAD

**Issue by joyngjr** [nus-cs2103-AY2223S2/forum#358]
If our UG/DG utilises collapsible sections to make it neater, are we allowed to mark out which sections are collapsible after we expand it and convert it to pdf? It isn't really obvious which parts belong to the collapsed section once we convert the documents to pdf.

**Comment by damithc** [comment]
@joyngjr What's the objective of doing so? 
In the PE, the UG is evaluated in the PDF format only. So, I imagine a feature available only in the Web versions is not relevant to the evaluation.

**Comment by joyngjr** [comment]
@damithc Hi prof, we added the collapsible section for UG because user-friendliness is a grading criteria under CS2101, and we didn't want our UG to look too cluttered. For DG, we added the collapsible section because we wanted to add in some auxiliary information which does not really fit in with the main part of the DG, but we thought could help developers in understanding our algorithm. 

Even though the feature is not available in the PDF format, can we retain it since it helps with the organisation of content?
